b2w-redirection.php

```diff
@@ -31,7 +31,7 @@ function rt_blogger_to_wordpress_add_option() {
 
 	wp_enqueue_style( 'rt-blogger-to-wordpress-redirection-css', ( RT_B2WR_PLUGIN_URL . 'css/b2w-redirection.css' ), array(), filemtime( RT_B2WR_PLUGIN_DIR . 'css/b2w-redirection.css' ) );
 
-	$page = filter_input( INPUT_GET, 'page', FILTER_SANITIZE_STRING );
+	$page = filter_input( INPUT_GET, 'page', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
```

Comment:
@pavanpatil1 Why we are not using `FILTER_SANITIZE_SPECIAL_CHARS` here?

Reply:
Hi @vishalkakadiya 
The official documentation recommends using `htmlspecialchars()` as `FILTER_SANITIZE_STRING` is deprecated.
`FILTER_SANITIZE_FULL_SPECIAL_CHARS` is equivalent to calling `htmlspecialchars()`

![image](https://github.com/rtCamp/blogger-to-wordpress/assets/8474784/b14b7722-9228-46cb-837b-31bdf5d17470)
![image](https://github.com/rtCamp/blogger-to-wordpress/assets/8474784/bf5c9ab6-27c9-4cfa-ba4c-00c084b3f73a)